nano/node/transport/udp.cpp

```diff
@@ -298,14 +317,26 @@ bool nano::transport::udp_channels::reserved_address (nano::endpoint const & end
 	return result;
 }
 
+nano::endpoint nano::transport::udp_channels::find_node_id (nano::account const & node_id_a)
```

Comment:
I think this should return the channel instead of the endpoint, the caller can always query for the endpoint if that's what they want.

Reply:
Updated